src/Shouldly/Shouldly.csproj

```diff
@@ -3,22 +3,18 @@
   <PropertyGroup>
     <Description>Shouldly - Assertion framework for .NET. The way asserting *Should* be</Description>
     <TargetFrameworks>net451;net40;netstandard1.3</TargetFrameworks>
+    <TargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">netstandard1.3</TargetFrameworks>
     <AssemblyName>Shouldly</AssemblyName>
     <PackageId>Shouldly</PackageId>
     <PackageTags>test;unit;testing;TDD;AAA;should;testunit;rspec;assert;assertion;framework</PackageTags>
     <PackageIconUrl>https://raw.github.com/shouldly/shouldly/master/Icons/package_icon.png</PackageIconUrl>
     <PackageProjectUrl>http://shouldly.github.com</PackageProjectUrl>
     <PackageLicenseUrl>https://github.com/shouldly/shouldly/blob/master/LICENSE.txt</PackageLicenseUrl>
-    <NetStandardImplicitPackageVersion>1.6.1</NetStandardImplicitPackageVersion>
```

Comment:
Haven't found the source of that remark.
I did find this:
https://github.com/shouldly/shouldly/pull/432#discussion_r125122510
That was mostly a objection to use `1.6.0` for the most part, I believe we get whatever the default for the sdk is, for 1.0.4 sdk = 1.6.1 implicit netstandard package. I think we should let it be implicit unless we understand a reason to specify it in order to not trip ourselves up in future.